lang/en.json

```diff
@@ -147,36 +147,36 @@
         "CustomVariables": "Custom variables",
         "CustomizeTracking": "Customize Tracking",
         "CustomizeTrackingTeaser": "Want to configure the Matomo tracking for this container? Here is a list of existing Matomo configurations in this container:",
-        "DataLayerVariableDescription": "Reads a custom value from the Data-Layer.",
-        "DataLayerVariableHelp": "Using this variable you can access any value that is stored within the dataLayer. You can also push values to the dataLayer yourself and access them this way afterwards.",
+        "DataLayerVariableDescription": "Reads a custom value from the data-layer.",
+        "DataLayerVariableHelp": "Using this variable you can access any value stored within the dataLayer. You can also push values to the dataLayer yourself and access them this way afterwards.",
         "DataLayerVariableName": "Data-Layer",
         "DefaultContainer": "Default Container",
         "DefaultValue": "Default value",
-        "DefaultValueHelp": "You can configure a default value which will be used if the variable does not return a value. Please note an empty string ('') is considered to be a value and will not fall back to the default value, configure a lookup value for this case if needed. Also note that the default value will be applied before the lookup table is evaluated.",
-        "DeleteContainerConfirm": "Are you sure you want to delete this container? You cannot undo the deletion. Please also make sure to remove all embedded code snippets for this container from your website as the container files will no longer be available after you have deleted this container.",
-        "DeleteTagConfirm": "Are you sure you want to delete this tag? You cannot undo the deletion.",
-        "DeleteTriggerConfirm": "Are you sure you want to delete this trigger? You cannot undo the deletion.",
-        "DeleteVariableConfirm": "Are you sure you want to delete this variable? You cannot undo the deletion.",
-        "DeleteVersionConfirm": "Are you sure you want to delete this version? You cannot undo the deletion.",
+        "DefaultValueHelp": "Configure a default value to use if the variable does not return a value. Please note an empty string ('') is considered to be a value and will not fall back to the default value, configure a lookup value for this case if needed. Also note that the default value will be applied before the lookup table is evaluated.",
```

Comment:
Here again we are missing the alert I feel

Reply:
What do you want to alert the user to?